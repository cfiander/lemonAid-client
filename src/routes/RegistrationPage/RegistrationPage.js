import React, { Component } from 'react'
import RegistrationForm from '../../components/RegistrationForm/RegistrationForm'

export default class RegistrationPage extends Component {
  static defaultProps = {
    history: {
      push: () => {},
    },
  }

  handleRegistrationSuccess = user => {
    const { history } = this.props
    history.push('/login')
  }

  render() {
    return (
      <section className="registration">
        <h1 className="heroTitle">Register</h1>
        <RegistrationForm
          onRegistrationSuccess={this.handleRegistrationSuccess}
        />
      </section>
    )
  }
}