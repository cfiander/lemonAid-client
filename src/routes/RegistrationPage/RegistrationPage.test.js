import React from 'react'
import { shallow } from 'enzyme'
import RegistrationPage from './RegistrationPage';
import toJson from 'enzyme-to-json'



describe('Renders the registration page',() =>{
    it('renders a registrationform by default', () => {
        const wrapper = shallow(<RegistrationPage />)
        expect(toJson(wrapper)).toMatchSnapshot()
      })
})