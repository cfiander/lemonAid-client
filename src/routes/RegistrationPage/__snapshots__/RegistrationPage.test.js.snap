// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the registration page renders a registrationform by default 1`] = `
<section
  className="registration"
>
  <h1
    className="heroTitle"
  >
    Register
  </h1>
  <RegistrationForm
    onRegistrationSuccess={[Function]}
  />
</section>
`;
